fe/src/shared/components/Files/Files.js

```diff
@@ -16,11 +16,10 @@ function Files(props) {
     };
 
     return (
-        
         <Fragment>
             {props.files &&
                 <section className="section files">
-                    <h3 className="title is-4">{props.downloadFilesLabel}</h3>
+                    <h3 className="title is-4">{props.isAttachments ? props.attachmentsLabel : props.downloadFilesLabel}</h3>
```

Comment:
**suggestion:** I'd rather avoid making it attachment-specific

Reply:
@dawiddworak88 

we should recognize files or attachments, for me is useless create new component. 

Or my second think is about change downloadLabel "Pliki do pobrania" to "Załączone pliki" and delete props isAttachments

Reply:
Hi @SebaKas21 , let's rename it to "Pliki" and remove all the "isAttachments" properties.